### PR TITLE
adds support in pipeline for voyage

### LINKED ISSUE #3423: bug/VOYAGE embedding models supported but not available in PIPELINE
**Describe the bug**
In Pipeline -> EmbedderConfig, every embedding model documented here
https://docs.unstructured.io/open-source/core-functionality/embedding#voyageaiembeddingencoder
is supported except for Voyage throws an error as being not recognized

**To Reproduce**
```
from unstructured.ingest.v2.pipeline.pipeline import Pipeline
from unstructured.ingest.v2.interfaces import ProcessorConfig
from unstructured.ingest.v2.processes.connectors.fsspec.s3 import (
    S3IndexerConfig,
    S3DownloaderConfig,
    S3ConnectionConfig,
    S3AccessConfig,
    S3UploaderConfig
)
from unstructured.ingest.v2.processes.partitioner import PartitionerConfig
from unstructured.ingest.v2.processes.chunker import ChunkerConfig
from unstructured.ingest.v2.processes.embedder import EmbedderConfig
pipeline = Pipeline.from_configs(
    context=ProcessorConfig(),
    indexer_config=S3IndexerConfig(remote_url=INPUT_S3_FILE),
    downloader_config=S3DownloaderConfig(download_dir="s3-ingest-download"),
    source_connection_config=S3ConnectionConfig(
        access_config=S3AccessConfig(
            key="AWS_ACCESS_KEY_ID",
            secret="AWS_SECRET_ACCESS_KEY",
            token="AWS_SESSION_TOKEN"
        )
    ),
    partitioner_config=PartitionerConfig(
        partition_by_api=True,
        api_key="UNSTRUCTURED_API_KEY_AUTH",
        partition_endpoint="UNSTRUCTURED_SERVER_URL",
        strategy="auto"
    ),
    chunker_config=ChunkerConfig(chunking_strategy="by_title",
                                chunk_combine_text_under_n_chars=100,
                                chunk_include_orig_elements=False,
                                chunk_max_characters=4000),
    embedder_config=EmbedderConfig(embedding_provider="Voyage",
                                   embedding_api_key="VOYAGE_API_KEY",
                                   embedding_model_name="voyage-law-2"),
    destination_connection_config=S3ConnectionConfig(
        access_config=S3AccessConfig(
            key="AWS_ACCESS_KEY_ID",
            secret="AWS_SECRET_ACCESS_KEY",
            token="AWS_SESSION_TOKEN"
        )
    ),
    uploader_config=S3UploaderConfig(remote_url=OUTPUT_S3_FILEPATH)
)
```

**Expected behavior**
Support for VoyageAIEmbeddingEncoder / Voyage to be a valid parameter
If support is not intended, there should be indication in the documentation that this is available functionality only when ran outside the pipeline

**Screenshots**

**Environment Info**
Python 3.11
ValueError: Voyage not a recognized encoder

**Additional context**

### PR DESCRIPTION
**Pull request by jeremydiba**
The following PR adds support for Voyage embedding models within the pipeline process.  

Fixes #3423

### CODE REVIEW

**Comment by jeremydiba** [comment]
What would be the best process of getting a review here?

File: unstructured/ingest/v2/processes/embedder.py [hunk 1]
```diff
@@ -58,6 +58,14 @@ def get_embedder(self) -> BaseEmbeddingEncoder:
             )
 
             return VertexAIEmbeddingEncoder(config=VertexAIEmbeddingConfig(**kwargs))
+
+        elif self.embedding_provider == "voyage":
```
Comment (christinestraub) [inline unstructured/ingest/v2/processes/embedder.py:62]: Please change "voyage" to "langchain-voyageai"? You can refer to: https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/ingest/interfaces.py#L237

**Comment by christinestraub** [comment]
https://github.com/Unstructured-IO/unstructured-ingest/blob/main/unstructured_ingest/v2/processes/embedder.py

**Comment by christinestraub** [comment]
Hi @jeremydiba, Thank you for your contribution.  We appreciate your contribution. However, I need to inform you that we have a new git repository for ingestion and this code seems to be already in there - https://github.com/Unstructured-IO/unstructured-ingest/blob/main/unstructured_ingest/v2/processes/embedder.py. The ingestion code in this repository is no longer under active development.

We are not accepting further changes to this codebase at this time. We apologize for any inconvenience this may cause, and we thank you for your understanding.

If you have any questions or would like to discuss this further, please don't hesitate to reach out.

**Comment by jeremydiba** [comment]
> Hi @jeremydiba, Thank you for your contribution. We appreciate your contribution. However, I need to inform you that we have a new git repository for ingestion and this code seems to be already in there - https://github.com/Unstructured-IO/unstructured-ingest/blob/main/unstructured_ingest/v2/processes/embedder.py. The ingestion code in this repository is no longer under active development.
> 
> We are not accepting further changes to this codebase at this time. Sorry for the inconvenience this may cause. If you have any questions or would like to discuss this further, please don't hesitate to reach out.

No problem, thanks for the response!

**Pull request closed without merging**